zzzz_modular_occulus/code/modules/cooking/seed.dm

```diff
@@ -0,0 +1,106 @@
+// Chili plants/variants.
+/datum/seed/chili
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SPICY_FOOD)
+
+/datum/seed/chili/ice
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SPICY_FOOD)
+
+/datum/seed/berry
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SWEET_FOOD)
+
+/datum/seed/berry/glow
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SWEET_FOOD, SPICY_FOOD)
+
+/datum/seed/berry/poison
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SWEET_FOOD, SPICY_FOOD)
+
+/datum/seed/berry/poison/death
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SPICY_FOOD, SALTY_FOOD)
+
+/datum/seed/nettle
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SPICY_FOOD)
+
+/datum/seed/nettle/death
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SPICY_FOOD, SALTY_FOOD)
+
+/datum/seed/tomato/blood
+	taste_tag = list(MEAT_FOOD)
+
+/datum/seed/tomato/killer
+	taste_tag = list(MEAT_FOOD)
+
+/datum/seed/tomato/blue
+	taste_tag = list(VEGETARIAN_FOOD, CHEESE_FOOD)
+
+/datum/seed/eggplant
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, UMAMI_FOOD)
+
+/datum/seed/apple
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SWEET_FOOD)
+
+/datum/seed/apple/poison
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, SWEET_FOOD, SPICY_FOOD)
+
+/datum/seed/ambrosia
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, UMAMI_FOOD)
+
+/datum/seed/mushroom
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, UMAMI_FOOD)
+
+/datum/seed/mushroom/plump
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, UMAMI_FOOD, SWEET_FOOD)
+
+/datum/seed/mushroom/poison
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, UMAMI_FOOD, SPICY_FOOD)
+
+/datum/seed/mushroom/glowshroom
+	taste_tag = list(VEGETARIAN_FOOD, VEGAN_FOOD, UMAMI_FOOD, SALTY_FOOD)
+
+/datum/seed/mushroom/maintshroom
```

Comment:
Other than this little intruder, and some whitespace on line 67-68, it looks fine.